index.html

```diff
@@ -1,15 +1,91 @@
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
+    <title>Segunda-Tarea</title>
+	<title>Mi Tienda Online</title>
 </head>
 <body>
-  <header></header>
-  <main></main>
-  <footer></footer>
-</body>
+  <header class="header">
+    <div class="header__top">
+      <div class="header__top-top container">
+        <div class="header__logo">
+          <img src="image/logo.png" alt=>
+        </div>
+        <div class="header__search">
+          <input class="header__search-input" type="text" placeholder="¿Qué buscas?">
+          <button class="header__search-button" class="fas fa-search">
+            <a href="" title="Buscar">
+              <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd" d="M7 12C9.76142 12 12 9.76142 12 7C12 4.23858 9.76142 2 7 2C4.23858 2 2 4.23858 2 7C2 9.76142 4.23858 12 7 12Z" stroke="#EEEEF7" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                <path d="M14.0001 14L10.5334 10.5333" stroke="#EEEEF7" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+              </svg>
+            </a>
+          </button>
+        </div>
+        <div class="header__location-hello">
+          <div class="header__location">
+            <img class="header__location-img" src="image/location.png" alt="location">Todas <div class="nav-links">
+              <a class="nav-link" href="#">Seleccionar tienda</a>
+            </div>
+          </div>
+          <div class="bord"></div>
+          <div class="header__hello">
+            <img class="header__hello-img" src="image/user.png" alt="hello">Hola! <div class="nav-links">
+              <a class="nav-link" href="#">Ingresa a tu cuenta</a>
+            </div>
+          </div>
+          <div class="bord"></div>
+          <a href="" title="Carrito">
+            <svg width="24" height="21" viewBox="0 0 24 21" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path fill-rule="evenodd" clip-rule="evenodd" d="M20.8401 2.60999C19.8086 1.57799 18.4093 0.998169 16.9501 0.998169C15.491 0.998169 14.0916 1.57799 13.0601 2.60999L12.0001 3.66999L10.9401 2.60999C8.79173 0.461607 5.30851 0.461607 3.16012 2.60999C1.01173 4.75838 1.01173 8.24161 3.16012 10.39L4.22012 11.45L12.0001 19.23L19.7801 11.45L20.8401 10.39C21.8721 9.35847 22.4519 7.95913 22.4519 6.49999C22.4519 5.04086 21.8721 3.64152 20.8401 2.60999Z" stroke="white" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+            </svg>
+          </a>
+          <div class="bord"></div>
+          <a href="" title="Seleccionar">
```

Comment:
Este si es el carrito
```suggestion
          <a href="" title="Carrito">
```